docs/markdown/clusters/running_jobs.md

```diff
@@ -24,7 +24,7 @@ of the functionality might not be available, for example the `cfc` and `binac` p
 
 Please start all your nextflow jobs from the head node.
 Nextflow interacts directly with the Slurm scheduler and will take care of submitting individual jobs to the nodes.
-If you submit via an interactive job, weird errors can occur, e.g. the cache directory for the containers is mounted read only on the compute nodes, so you can't pull new containers from a node.
+If you submit via an interactive job, strange errors can occur, e.g. the cache directory for the containers is mounted read-only on the compute nodes, so you can't pull new containers from a node.
 
 ### Dependencies
 
```

Comment:
This sentence would need to be changed: "Luckily, our sysadmin made a module for singularity and java in our clusters already, so you will just need to load these modules."

To sth like: "Java and singularity are already installed in all cluster nodes so these do not need to be installed."